webpay/pay/urls.py

```diff
@@ -1,39 +1,18 @@
-from django.conf import settings
-from django.conf.urls.defaults import patterns, url
+from django.conf.urls import patterns, url
 
-import views
+from .api import (callback_error_url, callback_success_url, PayViewSet,
+                  trans_start_url)
 
-urlpatterns = patterns('',
-    url(r'^$', views.index, name='pay.lobby'),
-    url(r'^configure-transaction$', views.configure_transaction,
-        name='pay.configure_transaction'),
-    # Be careful if you change this because it could be hard
-    # coded into settings. See settings.PAY_URLS.
-    url(r'^fake-bango-url$', views.fake_bango_url,
-        name='pay.fake_bango_url'),
-    url(r'^bounce$', views.bounce, name='pay.bounce'),
-    url(r'^simulate$', views.simulate, name='pay.simulate'),
-    url(r'^super_simulate$', views.super_simulate, name='pay.super_simulate'),
-    url(r'^wait_to_start$', views.wait_to_start, name='pay.wait_to_start'),
-    url(r'^trans_start_url$', views.trans_start_url,
+
+urlpatterns = patterns(
+    '',
+    url('^$',
+        PayViewSet.as_view({'post': 'create', 'get': 'retrieve'}),
+        name='pay'),
+    url(r'^trans_start_url$', trans_start_url,
         name='pay.trans_start_url'),
-    url(r'^callback_success_url$', views.callback_success_url,
+    url(r'^callback_success_url$', callback_success_url,
         name='pay.callback_success_url'),
-    url(r'^callback_error_url$', views.callback_error_url,
+    url(r'^callback_error_url$', callback_error_url,
         name='pay.callback_error_url'),
 )
-
-
-if settings.DEBUG:
```

Comment:
I think there might be some error templating (I thought I saw being deleting) but that might have been an old diff.